app/models/rental_unit.rb

```diff
@@ -10,4 +10,15 @@ def price
   def price_per_room
     price / rooms
   end
+
+  def total_applicable_price
+    if price_per_room > 5000
+      price + 500
+    end
+  end
+
+  def get_final_price
+    puts total_applicable_price
+    return total_applicable_price
```

Comment:
The `return` keyword is not necessary here. In Ruby, the last evaluated statement in a method is automatically returned.